server/services/return-of-service.js

```diff
@@ -2,6 +2,16 @@
 const { dbClient, collections } = require('../db');
 const { rosError } = require('../constants');
 
+const getRosSiteId = async (siteId) => {
```

Comment:
This naming is a bit confusing because it's get: `siteId` and we pass in `siteId`, would `getRosDbId` make more sense?

Reply:
Good point!